Woche6/dna/dna.py

```diff
@@ -0,0 +1,82 @@
+from csv import reader, DictReader
+from sys import argv, exit
+
+
+def main():
+
+    if len(argv) != 3:
+        print("Usage: python dna.py data.csv sequence.txt")
+        exit(1)
+
+    db_path = argv[1]
+    seq_path = argv[2]
+
+    with open(db_path, "r") as csvfile:
+        file_reader = DictReader(csvfile)
+        dict_list = list(file_reader)
+
+    with open(seq_path, "r") as file:
+        sequence = file.read()
+
+    sequence
+    max_counts = []
+    for i in range(1, len(reader.fieldnames)):
```

Comment:
```suggestion
    for i in range(1, len(file_reader.fieldnames)):
```
This should be the specific CsvReader that read the file, not any generic one.